examples/ref-implementation/backstage-templates/entities/app-with-bucket/skeleton/kustomize/prod/kustomization.yaml

```diff
@@ -0,0 +1,35 @@
+{%- if values.awsResources %}
```

Comment:
what do you think about splitting the example applications into a separate folder, so we have one folder for a vanilla reference implementation, and another to deploy the example apps. doing something like the following in the readme: 
```
idpbuilder create --package-dir examples/ref-implementation --package-dir examples/sample-apps
```

Reply:
I would rather just have one command to deploy everything.

Reply:
yeah +1 on what Charles said above.